app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -37,8 +37,16 @@ Fae.form.filtering = {
   _refreshTable: function() {
     // hardcode _this because this === Fryr object
     var _this = Fae.form.filtering;
-    var post_url = _this.$filter_form.attr('action');
-    var $results_table = ($(".js-results-table").length) ? $(".js-results-table").first() : _this.$filter_form.next('table');
+
+    if (_this.$filter_form.length) {
+      post_url = _this.$filter_form.attr('action');
+      $current_table = _this.$filter_form.next('table');
+    } else {
+      post_url = _this.$pagination.data('filter-path');
```

Comment:
The data attribute below is snake cased and this uses a hyphen. Is this working? I would assume it wouldn't.

Reply:
Yeah this works. Rails converts it to kebab case since symbols can't have hyphens. If I do `data: { filter-path: '' }` I get a syntax error